### PR TITLE
Checks namespace flags for user ns code path.

### PR DESCRIPTION
**Pull request by mrunalp**
Follow on to #324 

Signed-off-by: Mrunal Patel <mrunalp@gmail.com> (github: mrunalp)

### CODE REVIEW

**Comment by mrunalp** [comment]
@LK4D4 ping :)

File: namespaces/init.go [hunk 1]
```diff
@@ -223,6 +228,9 @@ func initUserNs(container *libcontainer.Config, uncleanRootfs, consolePath strin
 	}
 
 	if container.RestrictSys {
+		if (cloneFlags & syscall.CLONE_NEWNS) == 0 {
+			return fmt.Errorf("unable to restrict access to kernel files")
```
Comment (vmarmol) [inline namespaces/init.go]: nit: add "without mount namespaces"?

**Comment by vmarmol** [comment]
LGTM

File: namespaces/init.go (hunk 1 above)
Comment (mrunalp) [inline namespaces/init.go]: Fixing, thanks.

Comment (mrunalp) [inline namespaces/init.go]: Done.

**Comment by vmarmol** [comment]
LGTM, good to merge from my end :)

**Comment by LK4D4** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**